src/main/java/uk/co/fivium/dmda/server/SMTPConfig.java

```diff
@@ -236,13 +250,15 @@ else if (lDomainRegexNode != null && lDomainNode == null) {
             }
           }
 
-          lRecipientDatabaseMap.add(new DomainMatcher(lDomain, lIsRegexDomain, lDatabaseName));
+          lRecipientDatabaseMap.add(new DomainMatcher(lDomain, lIsRegexDomain, lDatabaseName, lPriority));
```

Comment:
Nitpick and I know you didn't add this, but can we rename `lRecipientDatabaseMap`? Is a bit confusing since it implies the datatype is a `Map` when its a `List`. e.g. `lRecipientDatabaseMappings`